doc/src/main/docs/features.asciidoc

```diff
@@ -1950,9 +1950,69 @@ IMPORTANT: Gentics Mesh by default returns the ```draft``` version of nodes. Thi
 
 NOTE: Reading published nodes requires the user to be assigned to a role which grants the ```readPublished``` permission.
 
+=== Listing versions
+
+Versions of a node can be listed via the ```/api/v1/:projectName/nodes/:uuid/versions``` endpoint.
+
+```
+"versions" : {
+    "de" : [ {
+      "creator" : {
+        "firstName" : "Joe",
+        "lastName" : "Doe",
+        "uuid" : "307cf98b66424db5bcf98b66424db5aa"
+      },
+      "created" : "2019-05-22T19:04:10Z",
+      "version" : "2.0",
+      "draft" : true,
+      "published" : true,
+      "branchRoot" : false
+    }
+    …
+  } ]
+}
+```
+
+A version can be used as a `draft` version. This version can also be referenced as a `published` version when published. The `branchRoot` flag indicates whether the version is being referenced as root version in a branch. All these three different kind of versions are not purgeable. Versions which have been made in betweet can be purged.
+
+NOTE: Versions are language specific. This means that each language of a node has its dedicated version history.
+
+[[controlling-versioning]]
+=== Controlling versioning
+
+Versioning can be controlled on a *per-schema* basis. The `versioned` property within the schema controls whether additional versions for a content will be kept.
+
+The default versioning flag from the `mesh.yml` configuration file will be used if for schemas which don't have `versioned` property set. Setting the `versioned` property will thus also allow to override the default configuration in the `mesh.yml`.
+
+The default behaviour can be configured via the environment variable `MESH_CONTENT_VERSIONING_ENABLED` 
+or the config file.
+
+.mesh.yml
+[source,yaml]
+----
+content:
+  versioning: false
+----
+
+NOTE: Disabling versioning will not stop version numbers to increase. Instead the mechanism will automatically purge the previously used version (if possible).
+
+=== Purging versions
+
+The ```POST /api/v1/projects/:projectUuid/maintenance/purge` endpoint can be used to purge older versions from the system.
+
+When invoked a new job will triggered which purges all possible content versions from the given project.
+
+The `?since` query parameter can be used to control what versions should be purged. 
+
+The call `POST /api/v1/projects/:projectUuid/maintenance/purge?since=2019-04-20T18:00Z` will thus only purge versions which are older than the given date. Omitting the parameter will purge all versions.
+
+Draft and publish versions will not be purged since they are still required. Additionally when the initial version of a node can also not be purged since it is bound to the _Branch_ of the node. Gentics Mesh requires this version to keep track of the branch starting point. The version lists endpoint will mark those versions using the `branchRoot` flag.
+
```

Comment:
* Add note that the version purge mechanism works independently from versioning enabled/disabled. 

* Add note that setting the versioning flag in the schema will not automatically purge versions.